clang/tools/clang-offload-wrapper/ClangOffloadWrapper.cpp

```diff
@@ -118,6 +122,13 @@ static cl::opt<std::string> Output("o", cl::Required,
                                    cl::value_desc("filename"),
                                    cl::cat(ClangOffloadWrapperCategory));
 
+static cl::opt<std::string>
```

Comment:
The inputs can be provided either in batch mode (.table) or as list of Input files in the command line. Does this list of SymPropBCFiles work in both cases? Also, I think I asked this above as well. Who is responsible for feeding this file to the wrapper?

Thanks

Reply:
-sym-prop-bc-files works in batch mode (see testcase clang-offload-wrapper-input-multi-bc.cpp) and non-batch mode (see testcase clang-offload-wrapper-input-bc.cpp).   In both cases, -sym-prop-bc-files specifies a file that has a list of sym/prop BC files.  The driver is responsible for creating this file.